pkg/networkmanager/bond.jsx

```diff
@@ -168,13 +168,13 @@ export const BondDialog = ({ connection, dev, settings }) => {
                     <MacMenu idPrefix={idPrefix} model={model} mac={mac} setMAC={setMAC} />
                 </FormGroup>
                 <FormGroup fieldId={idPrefix + "-mode-select"} label={_("Mode")}>
-                    <FormSelect id={idPrefix + "-mode-select"} onChange={setMode}
+                    <FormSelect id={idPrefix + "-mode-select"} onChange={(_, val) => setMode(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)